cmd/promtool/main.go

```diff
@@ -142,6 +142,10 @@ func main() {
 	dumpMinTime := tsdbDumpCmd.Flag("min-time", "Minimum timestamp to dump.").Default(strconv.FormatInt(math.MinInt64, 10)).Int64()
 	dumpMaxTime := tsdbDumpCmd.Flag("max-time", "Maximum timestamp to dump.").Default(strconv.FormatInt(math.MaxInt64, 10)).Int64()
 
+	tsdbAnalyzeCompaction := tsdbCmd.Command("analyze-compaction", "Plot a distribution of how full TSDB chunks are, relative to the maximum capacity of 120 samples.")
+	analyzeCompactionDB := tsdbAnalyzeCompaction.Arg("db path", "Database path (default is "+defaultDBPath+").").Default(defaultDBPath).String()
+	analyzeCompactionBlockID := tsdbAnalyzeCompaction.Arg("block ID", "The TSDB block to analyze").Default("").String()
```

Comment:
We could also consider adding this analysis to the existing `tsdb analyze` command.

Reply:
IMHO it would be fit in the scope of `tsdb analyze` but lets see what others think :)

Reply:
@dgl do you have any thoughts on this one?

Reply:
Yeah, makes sense to include it to me.

Reply:
+1 can we merge to tsdb analyze then? 🤗